### PR TITLE
Syncing configs from hosted production to the configs repo. Disabling…

### PR DESCRIPTION
**Pull request by derek2600hz**
… t38 by default. Shrinking send silence when idle to 1, but not negative. Adjusting order of video codecs to prefer VP8

### CODE REVIEW

File: freeswitch/vars/defaults.xml [hunk 1]
```diff
@@ -5,14 +5,15 @@
     
     <X-PRE-PROCESS cmd="set" data="ignore_display_updates=false"/>
     <X-PRE-PROCESS cmd="set" data="recordings_dir=/tmp/"/>
-    <X-PRE-PROCESS cmd="set" data="send_silence_when_idle=400"/>
-    <X-PRE-PROCESS cmd="set" data="sdp_m_per_ptime=false"/>
+    <X-PRE-PROCESS cmd="set" data="send_silence_when_idle=1"/>
+		<X-PRE-PROCESS cmd="set" data="sdp_m_per_ptime=false"/>
```
Comment (lazedo) [inline freeswitch/vars/defaults.xml:9]: please align

**Review by lazedo — COMMENTED** [review]
looks ok. how do you feel about moving the `recording` related vars to a `recording.xml` ?

File: freeswitch/vars/defaults.xml (hunk 1 above)
Comment (derek2600hz) [reply freeswitch/vars/defaults.xml]: Fixed!

**Comment by derek2600hz** [comment]
@lazedo I'm not against it. Do we have a lot more that we are going to add, or might want to change? If not, maybe we keep them in defaults.

**Pull request merged**